### PR TITLE
First public release

### PR DESCRIPTION
**Pull request by justinmills**

### CODE REVIEW

**Comment by justinmills** [comment]
Reviewers: @stevepescatore @cashman

**Comment by cashman** [comment]
Looks good.  Thanks, Justin!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+# DEPRECATED
+
+Activity on this repo has ceased. Future development should be
```
Comment (stevepescatore) [inline README.md]: This doesn't seem right.

**Comment by stevepescatore** [comment]
Looks good - just the one suggestion about the README.

File: README.md (hunk 1 above)
Comment (justinmills) [inline README.md]: Removed, squashed and force pushed.

**Pull request merged**